### PR TITLE
Corrected documentation spelling, typos, formatting. 

### PR DESCRIPTION
**Pull request by CodeCharm**
Very minor revisions for clarity. No code change.

### CODE REVIEW

**Comment by davidfowl** [comment]
@Eilon assigning this to you

File: src/Microsoft.Extensions.Logging.EventSource/LoggingEventSource.cs [hunk 1]
```diff
@@ -6,38 +6,34 @@
...
-    ///      | NAME : LEVEL            // specifies level for a particular logger (can have a * suffix).
+    ///     SPEC =                          // empty spec, same as *
+    ///          | NAME                     // Just a name the level is the default level
+    ///          | NAME : LEVEL             // specifies level for a particular logger (can have a * suffix).
     ///
-    /// Where Name is the name of a ILoggger (case matters), Name can have a * which acts as a wildcard
-    /// AS A SUFFIX.   Thus Net* will match any loggers that start with the 'Net'.
+    /// **NAME** is the name of a ILogger (case matters), Name can have a * which acts as a wildcard
+    /// *as a suffix*. Thus, `Net*` will match any loggers that start with the string 'Net'.
     ///
```
Comment (Eilon) [inline src/Microsoft.Extensions.Logging.EventSource/LoggingEventSource.cs]: These empty lines don't really do anything - they are ignored, just like in HTML.

To have paragraph breaks, each paragraph has to be wrapped in `<para>...</para>` tags, like this:

```xml
        /// <para>
        /// Para1
        /// </para>
        /// <para>
        /// Para2
        /// </para>
```

File: src/Microsoft.Extensions.Logging.EventSource/LoggingEventSource.cs [hunk 2]
```diff
@@ -6,38 +6,34 @@
 namespace Microsoft.Extensions.Logging.EventSource
 {
     /// <summary>
-    /// The LoggingEventSource is the bridge form all ILogger based logging to EventSource/EventListener logging.
+    /// The LoggingEventSource is the bridge form all `ILogger`-based logging to EventSource/EventListener logging.
```
[the remark this answers is not in the document]
Comment (CodeCharm) [reply src/Microsoft.Extensions.Logging.EventSource/LoggingEventSource.cs]: Good to know; will do.

[the remark this answers is not in the document]
Comment (Eilon) [reply src/Microsoft.Extensions.Logging.EventSource/LoggingEventSource.cs]: BTW sorry for taking so long to get back to you on this! We'll try to do better this time around!

[the remark this answers is not in the document]
Comment (CodeCharm) [reply src/Microsoft.Extensions.Logging.EventSource/LoggingEventSource.cs]: No worries. This is my first effort on improving docs by working from the xmldocs.

I was thrown by the https://docs.microsoft.com/en-us/contribute/ instructions which say that markdown is supported, but now I think that only applies to the non-reference pages (not built from xmldocs).

File: src/Microsoft.Extensions.Logging.EventSource/LoggingEventSource.cs (hunk 1 above)
Comment (CodeCharm) [reply src/Microsoft.Extensions.Logging.EventSource/LoggingEventSource.cs]: I've made the changes. b9094e57db52cb1bfc512b66246c4befcc467d09 has the final version.

Let me know if I still didn't understand what you meant, or if you think I'm still not understanding how the docs system pulls out the xmldoc and generates the reference page (which I believe shows up at https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.extensions.logging.eventsourcelogger)

**Comment by muratg** [comment]
We're closing this PR because no updates have been provided in a timely manner and we are unable to merge it.

We thank you for the contribution and look forward to collaborating more in the future.

**Pull request closed without merging**